app/test_helpers.go

```diff
@@ -136,31 +138,50 @@ func addTestAddrs(app *LiquidityApp, ctx sdk.Context, accNum int, accAmt sdk.Int
 	testAddrs := strategy(accNum)
 
 	initCoins := sdk.NewCoins(sdk.NewCoin(app.StakingKeeper.BondDenom(ctx), accAmt))
-	setTotalSupply(app, ctx, accAmt, accNum)
 
-	// fill all the addresses with some coins, set the loose pool tokens simultaneously
 	for _, addr := range testAddrs {
-		SaveAccount(app, ctx, addr, initCoins)
+		initAccountWithCoins(app, ctx, addr, initCoins)
 	}
 
 	return testAddrs
 }
 
+func initAccountWithCoins(app *LiquidityApp, ctx sdk.Context, addr sdk.AccAddress, coins sdk.Coins) {
+	err := app.BankKeeper.MintCoins(ctx, minttypes.ModuleName, coins)
+	if err != nil {
+		panic(err)
+	}
+
+	err = app.BankKeeper.SendCoinsFromModuleToAccount(ctx, minttypes.ModuleName, addr, coins)
+	if err != nil {
+		panic(err)
+	}
+}
+
```

Comment:
It seems it isn't different than `FundAccount` function above except for the panics. Maybe we can use `FundAccount` and panic if the returned error is not nil.

Reply:
I notice that `AddCoins` function has the same logic with `FundAccount`. `initAccountWithCoins` serves the same purpose with those functions but does not return error.

Reply:
Sure, Let me refactoring that, thanks!